using-mobile-extensions/adobe-target/README.md

```diff
@@ -357,136 +252,116 @@ purchasedProductIds.add("125");
 
 ```objectivec
-+ (void) clearPrefetchCache;
-```
+let mboxParameters = [
+"status": "Platinum"
+]
+let profileParameters = [
+"gender": "female"
+]
 
-#### Objective-C Example
+let product = ACPTargetProduct(id: "24D334", categoryId: "Stationary")
 
-```objectivec
-[ACPTarget clearPrefetchCache];
+let order = ACPTargetOrder(id: "ADCKKBC", total: NSNumber(value: 400.50), purchasedProductIds: ["34", "125"])
+
+let targetParameters = ACPTargetParameters(parameters: mboxParameters, profileParameters: profileParameters, product: product, order: order)
 ```
 {% endtab %}
 {% endtabs %}
 
-## Visual preview <a id="visual-preview"></a>
+### Merge behavior of Target parameters
 
-Visual preview mode allows you to easily perform end-to-end QA for Target activities by enrolling and previewing these activities on your device. This mode does not require a specialized testing set up. To get started, set up a URL scheme and generate the preview links. For more information, see [Target mobile preview](https://docs.adobe.com/content/help/en/target/using/implement-target/mobile-apps/target-mobile-preview.html).
+`TargetParameters` including `mboxParameters`, `profileParameters`, `orderParameters` and `productParameters` can be passed in Target APIs and also when creating `TargetPrefetch` or `TargetRequest` objects. The `TargetParameters` passed in public APIs are considered global parameters and are merged with the corresponding parameters in the individual `TargetRequest` or `TargetPrefetch` objects.
 
-You can also set an app deep link that can be triggered when selections are made in the preview mode by using the following methods:
+While merging, the new keys in the mbox parameters or the profile parameters are appended to the final dictionary while the keys with the same name are overwritten in each `TargetRequest` or `TargetPrefetch` object by those from global parameters. In the case or `TargetOrder` or `TargetProduct` objects, the object passed in global parameters shall replace the corresponding object in each of `TargetRequest` or `TargetPrefetch` objects.
 
-{% tabs %}
-{% tab title="Android" %}
-#### **Syntax**
+## Target Sessions
 
-```java
-public static void setPreviewRestartDeepLink(final Uri deepLink);
-```
+With Target release v2.1.4 iOS and  v1.1.3 Android, Target extension now supports persistent sessions. When a Target request is received, a new session ID is generated if one doesn't already exist and is sent in the request. This session ID along with the Edge Host returned from the Target are stored in the persistent storage for the configured `target.sessionTimeout`. If not configured, the default value for the session timeout is 30 minutes. These variables are reset and removed from persistent storage if no Target request is received during the configured `target.sessionTimeout` or if  [resetExperience API](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference#reset-user-experience) is called.
```

Comment:
Please reword as follows:
"The Target extension (version 2.1.4 for iOS) and  (version 1.1.3 for Android) now supports persistent sessions. When a Target request is received, if a session ID does not exist, a new ID is generated and is sent in the request. This ID, with the Edge Host that is returned from the Target, is kept in persistent storage for the configured `target.sessionTimeout`. If the timeout value is not configured, the default value is 30 minutes. If no Target request is received during the configured `target.sessionTimeout` or if  the [resetExperience](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference#reset-user-experience) API is called, these variables are reset and removed from persistent storage ."

(please don't include "API" in the link...only the actual API name should be in the link.)